cypress/e2e/gui/engageSphere.cy.js

```diff
@@ -0,0 +1,13 @@
+describe('Testes dos componentes da aplicação Engage Sphere', () => {
```

Comment:
Que tal só:

```suggestion
describe('EngageSphere', () => {
```

?